packages/sdk-v2/package.json

```diff
@@ -55,7 +56,7 @@
     "typescript": "^4.4.4"
   },
   "dependencies": {
-    "@gooddollar/goodprotocol": "^1.0.27",
+    "@gooddollar/goodprotocol": "file:.yalc/@gooddollar/goodprotocol",
```

Comment:
that shouldnt be

Reply:
i'll update when protocol is properly released